modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -689,8 +690,17 @@ class Session implements ISession {
         try {
             log.trace "Session > destroying"
             // shutdown thread pools
-            finalizePoolManager?.shutdown(aborted)
-            publishPoolManager?.shutdown(aborted)
+            try {
+                finalizePoolManager?.shutdown(aborted)
+                publishPoolManager?.shutdown(aborted)
+            }
+            catch( TimeoutException e ) {
+                final ignoreErrors = config.navigate('workflow.output.ignoreErrors', false)
```

Comment:
Was this flag introduced to handle this case? Wonder if [strict](https://github.com/nextflow-io/nextflow/blob/5577-fail-on-publish-timeout/modules/nextflow/src/main/groovy/nextflow/NF.groovy#L63-L63) mode should not be used instead?

Reply:
Actually the ignoring of publish errors can be controlled by either `nextflow.enable.strict` or `workflow.output.ignoreErrors`, so it would be consistent to check for both here.

I generally try to only use feature flags for language features though

Reply:
ok, just made some minor changes